src/EFCore.PG/Metadata/INpgsqlIndexAnnotations.cs

```diff
@@ -34,5 +34,13 @@ public interface INpgsqlIndexAnnotations : IRelationalIndexAnnotations
         /// http://www.postgresql.org/docs/current/static/sql-createindex.html
         /// </remarks>
         string Method { get; }
+
+        /// <summary>
+        /// The PostgreSQL index operators to be used.
+        /// </summary>
+        /// <remarks>
+        /// https://www.postgresql.org/docs/current/static/indexes-opclass.html
+        /// </remarks>
+        string[] Operators { get; }
```

Comment:
Immutable property with mutable collection. Is it fine in EF or should it be replaced by `IReadOnlyList`?

Reply:
Yes, the public interface should use `IReadOnlyList`.

Reply:
But the implementation has a setter. Should I just downcast to `string` and hope for the best? Or should I implement the property explicitly and have two of them?

Reply:
Went with the latter option 😄